blueocean/pom.xml

```diff
@@ -12,9 +12,15 @@
     <packaging>hpi</packaging>
 
     <name>Blue Ocean</name>
-    <url>https://wiki.jenkins-ci.org/display/JENKINS/Blue+Ocean+Plugin</url>
+    <url>https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean/doc/BlueOcean.adoc</url>
```

Comment:
Lots of other references in the other plugins here:
https://github.com/jenkinsci/blueocean-plugin/search?q=https%3A%2F%2Fwiki.jenkins-ci.org%2Fdisplay%2FJENKINS%2FBlue%2BOcean%2BPlugin&unscoped_q=https%3A%2F%2Fwiki.jenkins-ci.org%2Fdisplay%2FJENKINS%2FBlue%2BOcean%2BPlugin

I assume they should all point to this page?

Reply:
@timja I think that each of the plugins in this multi-plugin repository should have a separate page that documents the specific plugin.  Inside the plugin documentation page it could link to the Blue Ocean plugin so that readers can more easily navigate.